docs/source/migration_mt_django.rst

```diff
@@ -463,7 +463,12 @@ In the second section of this article, we introduced the fact that with citus, `
 After installing the library, changing the engine, and updating the models, run
 :code:`python manage.py makemigrations`. This will produce a migration to make the foreign keys composite when necessary.
 
-4. Distribute data in Citus
+.. warning::
+ If you use Citus 10, you will need to distribute the model inherited from "models.Model" as reference if you have a "ManyToMany" relationship 
```

Comment:
what about the other versions? Is this needed for versions > 10 ?

Reply:
It would be better if we more explicitly mention creating a reference table by providing an example command

Reply:
No in Citus 11 it is not required

Reply:
Actually there is a reference table in documentation
https://django-multitenant.readthedocs.io/en/latest/general.html#supported-django-and-citus-versions-pre-requisites